org.civicrm.bootstrapcivicrm/css/drop-down-arrow.svg

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
2 things about this svg:

1) I really don't like it being in the `css/` folder, as it obviously doesn't belong here. Would it be a problem to move it to a, say, `svgs/` folder?
2) Have you try running it source code through a a SVG optimizer tool like [this one](https://petercollingridge.appspot.com/svg-editor)? It simplifies the markup a lot and reduces the size of the file:
```svg
<svg xmlns:svg="http://www.w3.org/2000/svg" xmlns="http://www.w3.org/2000/svg" width="28" height="30" viewBox="0 0 28 30" id="svg2" version="1.1"><defs id="defs4"/><metadata id="metadata7"/><g id="layer2" transform="translate(-300.44186,-142.40416)" display="inline"/><g id="layer1" transform="translate(-300.44186,-142.40416)"><path id="path3386" d="m318.92 155.88q0 0.18-0.13 0.31l-3.04 3.04q-0.13 0.13-0.31 0.13-0.18 0-0.31-0.13l-3.04-3.04q-0.13-0.13-0.13-0.31 0-0.18 0.13-0.31 0.13-0.13 0.31-0.13l6.09 0q0.18 0 0.31 0.13 0.13 0.13 0.13 0.31z" fill="#777"/><path d="m300.84 142.4 0 30" id="path3405" style="fill:none;stroke-width:1;stroke:#777"/></g></svg>
```